packages/react-components/src/tag/src/TagList.tsx

```diff
@@ -59,7 +69,7 @@ function TagItem({
             {...mergeProps(
                 rest,
                 {
-                    variant: "outline",
+                    variant: "outline" as const,
```

Comment:
weird? I don't get the "as const" for this one

Reply:
it was a mistake